src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -203,61 +197,6 @@ private TagSpanIntervalTree<TTag> GetTagTree(ITextSnapshot snapshot, ImmutableDi
                     : new TagSpanIntervalTree<TTag>(snapshot.TextBuffer, _dataSource.SpanTrackingMode);
             }
 
-            private static bool TryStealTagsFromRelatedTagSource(TextContentChangedEventArgs e)
-            {
-                // see bug 778731
-#if INTERACTIVE
```

Comment:
@tmat Hmm, what does this ifdef even mean in EditorFeatures? Did we move this around and make a mistake somewhere?